tools/oeedger8r/CMakeLists.txt

```diff
@@ -31,6 +31,10 @@ add_custom_command(
   DEPENDS intel/Parser.mly)
 
 # Compile
+find_program(OCAMLOPT ocamlopt)
```

Comment:
Should we take this and the checks for ocamllex and ocamlyacc and do them in one place? Then we can do one if (NOT OR NOT OR NOT) with a single error message. Up to you.

Reply:
Yup that would look cleaner. I updated the commit.